MediaBrowser.Controller/SyncPlay/PlaybackRequests/PlayGroupRequest.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.SyncPlay;
+
+namespace MediaBrowser.Controller.SyncPlay
```

Comment:
```suggestion
namespace MediaBrowser.Controller.SyncPlay.PlaybackRequests
```

Namespace is incorrect for everything in a SyncPlay subfolder